public/facesmashmain.css

```diff
@@ -0,0 +1,105 @@
+*{
+	box-sizing: border-box;
+	font-size:24px;
```

Comment:
space between `font-size: 24px;`. Make sure you're _very_ intentional about always doing this.